portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,67 @@
+      <h1 id="header"><b>Pablo Dueñas' Portfolio:</b></h3>
+      <img id="mainImage" src="pablo-photos.JPG" alt="Photos of Pablo" style="width:500px;height:500px">
+      <hr>
+      <h3 id="header"><i>About me &#128515;:</i></h3>
+      <p id ="indent">My name is Pablo Dueñas, I am 21 years old, and I was born and raised in Los Angeles, California. I am an undergraduate student at California State 
+          Polytechnic University, Pomona majoring in Computer Science and minoring in Data Science. I am first-generation Mexican-American, 
+          first-generation college student, and a proud Angeleno. Growing up, I had always been fascinated by the evolution of technology and 
+          software. My first exposure to technology was in 2005 when my parents gifted me a Gameboy Advanced for Christmas. I was in awe as to 
+          how something so small could bring me such enjoyment. Flash forward to today, my dream is to become a software engineer and help improve 
+          the lives of many people through the power of code.</p>
+      <p id ="indent">As a child of immigrants from México, I carry a perspective that isn't very represented in the world of technology. 
+          I am able to empathize with the struggles many Latinos face while navigating the differences between their households and life in the United 
+          States. In the future, I would like to contribute back and create resources that will help Latinos and other marginalized groups improve their 
+          quality of life.</p>
+      <hr>
+      <h3 id="header"><i>Online profiles &#128100;:</i></h3>
+      <p style="text-align: center;"><a href="https://www.linkedin.com/in/pduenas007/">LinkedIn</a>            <a href="https://github.com/pabdue">GitHub</a></p>
+      <hr>
+      <h3 id="header"><i>List of projects &#128195;:</i></h3>
+      <dl>
+        <dt><b>Zodiac Generator</b></dt>
+        <dd>- Coded in Python</dd>
+        <dd>- Utilizes Tkinter module, exception handling, object-oriented programming</dd>
+        <dt><b>Username / Password Authentication</b></dt>
+        <dd>- Coded in Python</dd>
+        <dd>- Utilizes functions, dictionaries, exception handling</dd>
+        <dt><b>Combinatorial Calculator</b></dt>
+        <dd>- Coded in MIPS assembly language</dd>
+        <dd>- Utilizes subroutines, recursion, arrays</dd>
+        <dt><b>Gambling game</b></dt>
+        <dd>- Coded in MIPS assembly language</dd>
+        <dd>- Utilizes subroutines, recursion, arrays, loops </dd>
+        <dt><b>Cash register</b></dt>
+        <dd>- Coded in Java</dd>
+        <dd>- Utilizes object-oriented programming, exception handling, modules, arrays</dd>
+      </dl>
+      <hr>
+      <h3 id="header"><i>Fun facts &#128175;:</i></h3>      
+      <p>Click here to get a random <strong>fact</strong> about me: <button style="background-color: rgba(105, 105, 105, 0.205);" onclick="addRandomFact()">&#128272;</button></p>
       <div id="greeting-container"></div>
```

Comment:
Consider adding more space between the fun fact button and where the fun fact actually appears. On my screen, when I click it I have to scroll down to see it.